### PR TITLE
Bugfix for minicpmo vision test

### PR DESCRIPTION
**Pull request by yizhang2077**
## Motivation

It is a bugfix copied from https://github.com/sgl-project/sglang/pull/4969/commits/b5e87b7e6d68e601ad5c22f45c0ef14fc5a39eeb which will block ci, I just move it.
Co-auther @mickqian 
## Checklist

- [ ] Format your code according to the [Code Formatting with Pre-Commit](https://docs.sglang.ai/references/contribution_guide.html#code-formatting-with-pre-commit).
- [ ] Add unit tests as outlined in the [Running Unit Tests](https://docs.sglang.ai/references/contribution_guide.html#running-unit-tests-adding-to-ci).
- [ ] Update documentation / docstrings / example tutorials as needed, according to [Writing Documentation](https://docs.sglang.ai/references/contribution_guide.html#writing-documentation-running-docs-ci).
- [ ] Provide throughput / latency benchmark results and accuracy evaluation results as needed, according to [Benchmark and Profiling](https://docs.sglang.ai/references/benchmark_and_profiling.html) and [Accuracy Results](https://docs.sglang.ai/references/accuracy_evaluation.html).
- [ ] For reviewers: If you haven't made any contributions to this PR and are only assisting with merging the main branch, please remove yourself as a co-author when merging the PR.
- [ ] Please feel free to join our Slack channel at https://slack.sglang.ai to discuss your PR.

### CODE REVIEW

**Comment by mickqian** [comment]
LGTM

File: python/sglang/srt/managers/schedule_batch.py [hunk 1]
```diff
@@ -374,6 +374,12 @@ def merge(self, other: MultimodalInputs):
                 self.mrope_position_delta = torch.cat(
                     [self.mrope_position_delta, other.mrope_position_delta], dim=0
                 )
+
```
Comment (hnyls2002) [inline python/sglang/srt/managers/schedule_batch.py:377]: only consider `unk_id`?

**Review by hnyls2002 — APPROVED** [review]

File: python/sglang/srt/managers/schedule_batch.py (hunk 1 above)
Comment (yizhang2077) [reply python/sglang/srt/managers/schedule_batch.py:377]: I think like `im_token_id` is also be considered, but it is a tricky method

**Comment by YosanHo** [comment]
@yizhang2077 @mickqian  After merge this commit, when using a single node with 8*H20, "not enough memory" error occurred when deploy DeepSeek-V3. this is a breaking change, please change it as a optional configuration
